### ISSUE TITLE
Resize-able / exchangeable frames 

### ISSUE THREAD

**Issue by trochumski** [romanvm/python-web-pdb#19]
Would it be possible to implement handles to resize frames (e.g. decrease height of "PDB Console") ?
Or to switch places of two frames.

I usually focus more on the code than on PDB output :)

**Comment by romanvm** [comment]
I agree that the UI needs some changes, but currently I have no time for anything significant.
